### ISSUE TITLE
Active, accepted findings getting mitigated when reimporting

### ISSUE THREAD

**Issue by gietschess** [DefectDojo/django-DefectDojo#8797]
**Be informative**
This issue relates to mr #7447.
Risk accepted Findings are set to mitigated even if they still exists in the imported scan.

**Bug description**
We're mostly working with trivy findings in defectdojo.
When we're accepting a finding with the simple risk acceptance function the finding gets the status 'Inactive', 'Risk Accepted'.
After the next reimport it also gets the status 'Mitigated', even if the active finding still exists in the report.

**Steps to reproduce**
1. Import findings of type 'Trivy Scan' via the reimport api
2. Go to http://localhost:8080/test/some-test-id, click on the menu points in front of one finding and hit 'Accept Risk' ('Enable simple risk acceptance' has to be enabled in the products settings to do this)
4. Reimport the same findings of type 'Trivy Scan' again via the reimport api
5. See that the previously accepted Finding got the additional status 'Mitigated'

**Expected behavior**
The finding should be skipped by the import (which it correctly does).
The finding should get **not** the status 'Mitigated' later in the reimport process when its still active.

In the Trivy Scan parser there are no informations for false positive, out of scope or risk accepted so these fields are set to False for each finding by default. By that the check mentioned in the [merge request](https://github.com/DefectDojo/django-DefectDojo/pull/7447) will never be True and the finding won't be added to the list of unchanged findings.
Without that the finding will be added to the to_mitigate list later.
See:
[dojo/importers/reimporter/reimporter.py#L431-L433](https://github.com/DefectDojo/django-DefectDojo/blob/bef8f1483ca8be21d658073dcfb4070ed2f173a0/dojo/importers/reimporter/reimporter.py#L431-L433)
[dojo/importers/reimporter/reimporter.py#L507](https://github.com/DefectDojo/django-DefectDojo/blob/bef8f1483ca8be21d658073dcfb4070ed2f173a0/dojo/importers/reimporter/reimporter.py#L507)

**Deployment method**
- [ ] Docker Compose
- [X] Kubernetes
- [ ] GoDojo

**Environment information**
 - DefectDojo version: 2.27.0
 - Trivy version: 0.45.1

**Logs** 
[uwsgi.log](https://github.com/DefectDojo/django-DefectDojo/files/12819937/uwsgi.log)
```
[05/Oct/2023 14:01:11] DEBUG [dojo.importers.reimporter.reimporter:123] 0: skipping existing finding (it is marked as false positive:False and/or out of scope:False or is a risk accepted:True): 3415:CVE-2023-38039 Curl 8.2.1-r0:curl:8.2.1-r0
[...]
[05/Oct/2023 14:01:11] DEBUG [dojo.importers.reimporter.reimporter:530] mitigating finding: 3415:CVE-2023-38039 Curl 8.2.1-r0
```
**Sample scan files**
[trivy-output.json.txt](https://github.com/DefectDojo/django-DefectDojo/files/12819892/trivy-output.json.txt)

**Reimport cURL**
[reimport.curl.txt](https://github.com/DefectDojo/django-DefectDojo/files/12819817/reimport.curl.txt)

**Comment by coheigea** [comment]
I'll take a look at this next week

**Comment by gietschess** [comment]
fixed with 2.38.0. thanks @lme-nca
